projects/packages/forms/src/abilities/class-forms-abilities.php

```diff
@@ -0,0 +1,455 @@
+			$request->set_param( 'include', $args['ids'] );
+		}
+
+		$response = $endpoint->get_items( $request );
+		if ( is_wp_error( $response ) ) {
+			return $response;
+		}
+
+		return $response->get_data();
+	}
+
+	/**
+	 * Update form submission callback.
+	 *
+	 * @param array $args Arguments from the ability input.
+	 * @return array|\WP_Error Returns updated submission data or WP_Error on failure.
+	 */
+	public static function update_form_submission( $args ) {
+		if ( ! isset( $args['id'] ) ) {
+			return new \WP_Error( 'missing_id', __( 'Submission ID is required.', 'jetpack-forms' ) );
+		}
+
+		$endpoint = new Contact_Form_Endpoint( 'feedback' );
+		$result   = array();
+
+		// Update status if provided
+		if ( isset( $args['status'] ) ) {
+			$request = new \WP_REST_Request( 'POST', '/wp/v2/feedback/' . $args['id'] );
+			$request->set_url_params( array( 'id' => $args['id'] ) );
+			$request->set_body_params( array( 'status' => $args['status'] ) );
+
+			$response = $endpoint->update_item( $request );
+			if ( is_wp_error( $response ) ) {
+				return $response;
+			}
+			$result = $response->get_data();
+		}
+
+		// Update read status if provided
+		if ( isset( $args['is_unread'] ) ) {
+			$request = new \WP_REST_Request( 'POST', '/wp/v2/feedback/' . $args['id'] . '/read' );
+			$request->set_url_params( array( 'id' => $args['id'] ) );
+			$request->set_body_params( array( 'is_unread' => $args['is_unread'] ) );
+
+			$response = $endpoint->update_read_status( $request );
+			if ( is_wp_error( $response ) ) {
+				return $response;
+			}
+			$result = array_merge( $result, $response->get_data() );
+		}
+
+		return $result;
+	}
+
+	/**
+	 * Delete form submission callback.
+	 *
+	 * @param array $args Arguments from the ability input.
+	 * @return array|\WP_Error Returns deletion result or WP_Error on failure.
+	 */
+	public static function delete_form_submission( $args ) {
+		if ( ! isset( $args['id'] ) ) {
+			return new \WP_Error( 'missing_id', __( 'Submission ID is required.', 'jetpack-forms' ) );
+		}
+
+		$endpoint = new Contact_Form_Endpoint( 'feedback' );
+		$request  = new \WP_REST_Request( 'DELETE', '/wp/v2/feedback/' . $args['id'] );
+		$request->set_url_params( array( 'id' => $args['id'] ) );
+		$request->set_param( 'force', true );
+
+		$response = $endpoint->delete_item( $request );
+		if ( is_wp_error( $response ) ) {
+			return $response;
+		}
+
+		return $response->get_data();
+	}
+
+	/**
+	 * Get status counts callback.
+	 *
+	 * @param array $args Arguments from the ability input.
+	 * @return array|\WP_Error Returns status counts or WP_Error on failure.
+	 */
+	public static function get_status_counts( $args = array() ) {
+		$args     = is_array( $args ) ? $args : array();
+		$endpoint = new Contact_Form_Endpoint( 'feedback' );
+		$request  = new \WP_REST_Request( 'GET', '/wp/v2/feedback/counts' );
+
+		self::set_params_from_args(
+			$request,
+			$args,
+			array( 'search', 'parent', 'before', 'after', 'is_unread' )
```

Comment:
Could this list be shared somewhere, so that in future when we expand, we don't remember to update two places? It's a few times in this file too so could consolidate within this one file.